contracts/axelar-gateway/src/contract.rs

```diff
@@ -9,6 +9,15 @@ use crate::{auth, event};
 
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+/// Parameters for extending the contract instance and its instance storage.
+/// 
+/// If the instance's time to live falls below 7 days, it will be extended by 60 days.
+/// 
```

Comment:
Linting test failed, please check on it

Reply:
The linter does not like the extra `///` for blank lines. However, this is how multi-line comments appear in the rest of the contract. Do we allow it, or do we remove all extra `///`?